scripts/template_upload_all.py

```diff
@@ -62,102 +62,124 @@ def parse_cmd_line():
     return args
 
 
-def template_name(image_link, image_ts, checksum_link, version=None):
+def template_name(image_link, timestamp, version=None):
+    """Generate a template name from given link, a timestamp, and optional version string
+    This method should handle naming templates from the following URL types:
+        - http://file.cloudforms.lab.eng.rdu2.redhat.com/builds/manageiq/master/latest/
+        - http://file.cloudforms.lab.eng.rdu2.redhat.com/builds/manageiq/gaprindashvili/stable/
+        - http://file.cloudforms.lab.eng.rdu2.redhat.com/builds/manageiq/fine/stable/
+        - http://file.cloudforms.lab.eng.rdu2.redhat.com/builds/cfme/5.7/stable/
+        - http://file.cloudforms.lab.eng.rdu2.redhat.com/builds/cfme/5.9/latest/
```

Comment:
Can you please replace the hostname with a placeholder so it is not exposed much?

Reply:
absolutely, my bad.